crates/nu-utils/standard_library/test_logger.nu

```diff
@@ -0,0 +1,40 @@
+use std.nu assert
+use logger.nu *
+
+def "assert no message" [output: string] {
+    assert (($output | describe) == "nothing")
+}
+
+def "assert message" [level_str: string, output:string] {
+    assert ($output | str contains $level_str)
+    assert ($output | str contains "test message")
+}
+
+export def test_critical [] {
+    assert no message (NU_LOG_LEVEL=99 log critical  "test message")
+    assert message CRIT (NU_LOG_LEVEL=CRITICAL log critical "test message")
+}
+export def test_error [] {
+    assert no message (NU_LOG_LEVEL=CRITICAL log error "test message")
+    assert message ERROR (NU_LOG_LEVEL=ERROR log error "test message")
+}
+export def test_warn [] {
+    assert no message (NU_LOG_LEVEL=ERROR log warning "test message")
+    assert message WARN (NU_LOG_LEVEL=WARNING log warning "test message")
+}
+export def test_info [] {
+    assert no message (NU_LOG_LEVEL=WARNING log info "test message")
+    assert message INFO (NU_LOG_LEVEL=INFO log info "test message")
+}
+export def test_debug [] {
+    assert no message (NU_LOG_LEVEL=INFO log debug "test message")
+    assert message DEBUG (NU_LOG_LEVEL=DEBUG log debug "test message")
+}
+
+export def example_log [] {

```

Comment:
i think you can remove this, it was only for the example, right?

'cause there are no `assert`s in it :thinking:

Reply:
oh yeah i love examples, do not get me wrong :star_struck: 

then maybe above the definitions of the actual commands?
e.g.
```bash
# Log debug message
#
# # Example
# ```nushell
# >_ use /path/to/std.nu
# >_ std log debug "this is a debug message"
# DEBUG  this is a debug message
# ```
export def "log debug" [message: string] {
    if (current-log-level) > (DEBUG_LEVEL) { return }
    print --stderr $"(ansi default_dimmed)DEBUG ($message)(ansi reset)"
}
```
?

i think the real advantage here is that it would show up in `help std log debug` :yum: